environments.json

```diff
@@ -1497,22 +1497,30 @@
     "short": "CH&nbsp;61,<br>OP&nbsp;48",
     "release": "2017-09-05",
     "note_id": "experimental-flag",
-    "obsolete": false
```

Comment:
I'm not comfortable with v61 being marked obsolete just because v62 was released.

Reply:
Me too

Reply:
This is how this was done so far.  Ditto for Firefox.

Reply:
I mean, if we want to change how things work, it should be a separate PR for all browsers, not just Chrome.